go_optouts/app.py

```diff
@@ -0,0 +1,81 @@
+#--------------------------------Hello World----------------------------------------------------
+from klein import Klein
+app = Klein()
+
+@app.route('/Hello')
+def hello(request):
+    return 'Hello, world!'
+
+
+#-----------------------------------------Adding Routes----------------------------------------------
+
+@app.route('/AR')
+def pg_root(request):
+	return 'I am the root page!'
+
+#@app.route('/about')
+#def pg_about(request):
+		#return 'I am Klein application'
+
+#app.run("localhost", 8080)
+
+#-----------------------------------------Variable Routes----------------------------------------------
+
+@app.route('/VR/<username>')
+def pg_VR(request, username):
+    return 'Hi %s!' % (username,)
+
+
+#from klein import Klein
+#app = Klein ()
+
+#@app.route('/<string:arg>')
+#def pg_string(request, arg):
+	#return 'String: %s!' %(arg,)
+
+#@app.route('/<int:arg>')
+#def pg_int(request, arg):
+	#return 'int: %s!' %(arg,)
+#@app.route('/<float:arg>')
+#def pg_float(request, arg):
+	#return 'Float: %s!' %(arg,)
+
+#app.run("localhost", 8080) 
```

Comment:
We try not to leave commented out code in our files because its often unclear to people who will read it later what the purpose of the commented out code is. Commented out code is also untested code and unused code, neither of which is particularly great either.